### ISSUE TITLE
/projects/source API does not support gpgkey_urls

### ISSUE THREAD

**Issue by bcl** [osbuild/osbuild-composer#593]
Similar to issue #592, the store doesn't support a gpgkey_urls list, so in the API this field is ignored.
This will require changes to the store.

**Comment by teg** [comment]
These changes would be forward-compatible, right? So this could be added?

**Comment by teg** [comment]
One more thing: Would we need to pass these paths down to osbuild, or could we extract the gpg keys and pass them using the existing API?

**Comment by bcl** [comment]
Yeah, since it's an addition they'll just be empty when reading the old store. Since we now how remote workers they should be passed to them as the actual key, not a path on the host.
